src/services/connections.js

```diff
@@ -82,6 +88,31 @@ module.exports = class ConnectionHelper {
 				})
 			}
 
+			const defaultOrgId = await getDefaultOrgId()
```

Comment:
@nevil-mathew we can skip making an API call here if we just need default org id

Reply:
Updated `getDefaultOrgId()` to return from the environment if it exists.